docs/guides/elasticstack-and-cloud.md

```diff
@@ -22,12 +22,10 @@ terraform {
 
   required_providers {
     ec = {
-      source  = "elastic/ec"
-      version = "~>0.3.0"
+      source = "elastic/ec"
     }
     elasticstack = {
-      source  = "elastic/elasticstack"
-      version = "~>0.5.0"
```

Comment:
Wonder if it would be better to have `version = "~>0.5"` here so we're requiring a provider version that will work with the example?

Reply:
Yeah that makes sense